### ISSUE TITLE
lximage-qt: broken dependency

### ISSUE THREAD

**Issue by misaka4e21** [AOSC-Dev/aosc-os-abbs#2046]
Bug description
-----------------------

lximage-qt: error while loading shared libraries: `libfm-qt.so.5`: cannot open shared object file: No such file or directory

`dpkg -L libfm-qt`:

    ...
    /usr/lib/libfm-qt.so.6

Reproducing steps
---------------------------

Install lximage-qt and run it.

Update channel(s)
--------------------------

stable

Package and version
------------------------------

```
dpkg -s lximage-qt 
Package: lximage-qt
Status: install ok installed
Section: x11
Installed-Size: 528
Maintainer: Null Packager <null@aosc.xyz>
Architecture: amd64
Version: 0.7.0
Depends: libexif (>= 0.6.21-4), pcmanfm-qt (>= 0.13.0)
Description: A lightweight image viewer and screenshot program
```

**Comment by MingcongBai** [comment]
@misaka4e21 The sover on libfm-qt.so has changed from 5 to 6, marking PKGBREAK and rebuilding relevant packages.

**Comment by MingcongBai** [comment]
Addressed as of LxQt 0.16 (with the two commits above). Closing.
